src/components/operations/operation-details/ko/runtime/authorization.ts

```diff
@@ -380,8 +384,8 @@ export class Authorization {
             return;
         }
 
-        if (this.api().type === TypeOfApi.webSocket) {
-            this.setSubscriptionKeyParameter(subscriptionKey)
+        if (this.api().type === TypeOfApi.webSocket || this.isGraphQL()) {
+            this.setSubscriptionKeyParameter(subscriptionKey);
```

Comment:
I see you moved the subscription-key from headers to parameters. But in the GraphQL console, the parameters are not shown as a separate section. Is this the expected behavior?

Reply:
This is something we haven't discussed before, but I think we definitely want to support this in the GraphQL console. All set, added. Thanks a ton for pointing this out.